modules/javafx.controls/src/main/java/javafx/scene/control/Spinner.java

```diff
@@ -845,6 +845,15 @@ public Object queryAccessibleAttribute(AccessibleAttribute attribute, Object...
                 }
                 return value != null ? value.toString() : "";
             }
+
+            case TEXT: {
+                String accText = getAccessibleText();
+                return (accText != null) ? accText : "";
```

Comment:
Is there a better default than the empty string? What do we do for Slider and other similar controls?

Reply:
With this fix, a11y client app reads a spinner as,
- when accessible text is set: [Accessible Text] Spinner [Value] [Editable / read only]
- when accessible text is not set: Spinner [Value] [Editable / read only]

Other controls like ComboBox, ChoiceBox return the value of control when accessible text is not set.
If we do the same for spinner, then the Spinner gets read as:  [Value] Spinner [Value] [Editable / read only]
In this case the value gets read twice.

The way spinner is read when accessible text is not set, it seems Ok to skip accessible text when it is not set.